core/src/com/mygdx/l5rdraft/packs/Card.java

```diff
@@ -48,7 +57,43 @@ public boolean equals(Object obj) {
         return false;
     }
 
+    @Override
+    public int hashCode() {
+        int hash = 7;
+        hash = 31 * hash + name.hashCode();
+        return hash;
+    }
+
```

Comment:
hashCode will need more work later, but it's probably good to have. Right now, if you have like conflict Tadaka and dynasty Tadaka, they would be considered the same object with your equals method. Suggestion is to add an id field to the card class.